repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -252,6 +257,55 @@
         </xs:simpleContent>
     </xs:complexType>
 
+    <xs:complexType name="ViaConfiguration">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    Contains the information needed to configure Repose's Via header.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:simpleContent>
+            <xs:extension base="papic:EmptyString">
+                <xs:attribute name="enabled" type="xs:boolean" use="optional" default="true">
+                    <xs:annotation>
+                        <xs:documentation>
+                            <html:p>Indicates whether or not the Via header is created.</html:p>
+                        </xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="content" type="xs:string" use="optional">
```

Comment:
The `content` attribute was intended to be the user supplied portion of the `Via` header like it is now. I am definitely open to a new name.

Is it supposed to only disable the version? The confusion seems to be here:
> New element will have an optional boolean attribute that defaults to true to enable/disable the *Via header entirely*

and

> New element will have an optional String attribute to specify the prefix of the Via header
>  - no prefix and *repose value* is off, no via header will be present

(emphasis added by me)